json/visitor.go

```diff
@@ -433,7 +432,7 @@ func (vs *Visitor) onFloat(f float64, bits int) error {
 }
 
 func (s *boolStack) init() {
-	s.stack = s.stack0[:0]
+	s.stack = make([]bool, 0, 32)
```

Comment:
could you add a comment as to why 32 is the initial capacity?

Reply:
That is hard to tell tbh. The original code used to use `stack0 [32]bool`.
To my best knowledge so far, there was no issue with this original sizing so I continued to use it. But I can not tell why the original capacity was chosen.
As I want to avoid a comment in the style `there used to be something with this capacity` how should this be documented?